components/location/listItem.jsx

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable react/prop-types */
+import React, { useEffect, useState } from "react";
+
+const LocationListItem = ({
```

Comment:
Would it make sense to make this `LocationCard` or do you think `LocationListItem` is clearer?

Reply:
Don't feel strongly either way, just wanted to use existing terminology here. I do think I like `LocationCard` better so long as we're clear about the distinction between a "card" and a "popup" (i.e. the info in the map marker)!